docs/configuring-polaris-for-production.md

```diff
@@ -55,6 +55,13 @@ Be sure to secure your metastore backend since it will be storing credentials an
 
 To use [EclipseLink](https://eclipse.dev/eclipselink/) for metastore management, specify the configuration `metaStoreManager.conf-file` to point to an [EclipseLink `persistence.xml` file](https://eclipse.dev/eclipselink/documentation/2.5/solutions/testingjpa002.htm). This file, local to the Polaris service, will contain information on what database to use for metastore management and how to connect to it.
 
+> [!IMPORTANT]
+> To use EclipseLink, you need to ensure that you have done two things:
```

Comment:
A few nits,

Avoid "you" - EclipseLink requires:

2. Set the `eclipseLink` gradle property to true within the Polaris build.

Reply:
Done!